nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-api/src/main/java/org/apache/nifi/web/controller/ControllerSearchService.java

```diff
@@ -175,13 +176,67 @@ public void search(final SearchResultsDTO results, final String search, final Pr
             }
         }
 
```

Comment:
Only question i have for the reviewer is if the `getControllerServices()` call here should be recursive.